src/modules/displaymeals.js

```diff
@@ -0,0 +1,19 @@
+// create function to display meal list dynamically
+const mealListDisplay = (meal) => {
+  const container = document.getElementById('meals-container');
+  const div = document.createElement('div');
+  // div section for meal card and info
+  div.setAttribute('data-id', meal.idCategory);
+  // div.innerHTML = `
+  // <div class="meal-card">
+  // <div class="img-of-meal"><img src="${meal.strCategoryThumb}" alt="meal img"></div>
+  // <div class="meal-title-name">
+  // <div class="meal-title"><h3>${meal.strCategory}</h3></div>
+  // <div class="likes-display"><i class="uil uil-heart love-icon"></i><p>Likes</p></div>
+  // </div>
+  // <button type="button" class="comment-Button" id="comment-button">Comment</button></div>`;
+
```

Comment:
Hi @jorgegoco ,

You did a great job so far!👏
You've done almost all of the requested changes, but there is a need for some improvements.
![image](https://user-images.githubusercontent.com/90766123/205445038-e8d9a20c-ec16-4643-b41d-57a5bc116e90.png)

almost there!

**Highlights 💯
No linter errors. ✔️
GitHub flow is followed. ✔️
PR has a good title and summary. ✔️
Asynchronous functions have been used ✔️
Beautiful design ✔️**

### Required Changes ♻️
Please remove all unused commented-out code from your displaymeals.js file

```suggestion
  div.innerHTML = `
  <div class="meal-card">
  <div class="img-of-meal"><img src="${meal.strCategoryThumb}" alt="meal img"></div>
  <div class="meal-title-name">
  <div class="meal-title"><h3>${meal.strCategory}</h3></div>
  <div class="likes-display"><i class="uil uil-heart love-icon"></i><p>Likes</p></div>
  </div>
  <button type="button" class="comment-Button" id="comment-button">Comment</button></div>`;

```

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).